pkg/operator/sync.go

```diff
@@ -20,12 +21,18 @@ import (
 )
 
 const (
-	deploymentRolloutPollInterval     = time.Second
-	deploymentRolloutTimeout          = 5 * time.Minute
-	deploymentMinimumAvailabilityTime = 3 * time.Minute
-	daemonsetRolloutPollInterval      = time.Second
-	daemonsetRolloutTimeout           = 5 * time.Minute
-	machineAPITerminationHandler      = "machine-api-termination-handler"
+	deploymentRolloutPollInterval       = time.Second
+	deploymentRolloutTimeout            = 5 * time.Minute
+	deploymentMinimumAvailabilityTime   = 3 * time.Minute
+	daemonsetRolloutPollInterval        = time.Second
+	daemonsetRolloutTimeout             = 5 * time.Minute
+	machineAPITerminationHandler        = "machine-api-termination-handler"
+	machineExposeMetricsPort            = 8041
+	machineSetExposeMetricsPort         = 8042
+	nodelinkExposeMetricsPort           = 8043
+	machineHealthCheckExposeMetricsPort = 8044
```

Comment:
Traditionally, the suggestions tend to be `8080` and `8443` for http/https respectively, it obviously doesn't really matter to us, but perhaps these should be `8441`, `8442`, `8443` and `8444`, Using that first 4 to denote that it's HTTPs ish 🤷  WDYT?

Reply:
Can change, SGTM